fuse_core/package.xml

```diff
@@ -17,6 +17,7 @@
   <depend>fuse_msgs</depend>
   <depend>pluginlib</depend>
   <depend>roscpp</depend>
+  <depend version_gte="1.13.8">rosconsole</depend>
```

Comment:
I didn't even know specifying versions was a thing. Thanks for that.

Reply:
FYI https://www.ros.org/reps/rep-0140.html#build-depend-multiple

It only mentions `build_depend`, but I've seen it used in all _depend_ flavours.

BTW I tried to confirm it doesn't break anything, but `rospack depends1 fuse_core` didn't work in my system. It complained about not finding other dependencies, like `libceres-dev` or even `boost`. I'm pretty sure this is only in my system. I know other packages using this w/o issues for a while, going through the entire debian generation process.

I guess the automatic checks you have in this repo should guarantee it's fine, or maybe you need to run an additional test to confirm it's well supported in your infrastructure.